Climate_police/tests/hw7-mao/pollution_map.py

```diff
@@ -0,0 +1,44 @@
+import plotly.offline as py
+py.init_notebook_mode()
```

Comment:
This is needed only to print the figures in a notebook! you shouldn't need this if you are creating JPEG files using the script.